stdlib/2/popen2.pyi

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Iterable, List, Optional, Union, Tuple, TypeVar
+
+_T = TypeVar('_T')
+
+__all__ = ...  # type: List[str]
+
+class Popen3:
+    sts = ...  # type: int
+    cmd = ...  # type: Iterable
+    pid = ...  # type: int
+    tochild = ...  # type: file
```

Comment:
Consider using `typing.TextIO` for `tochild` and `fromchild`, since `typing.IO` is what's returned from `fdopen`, and the `mode` parameter to `fdopen` doesn't include `b'. (So it's a TextIO, not a BinaryIO)

Also `childerr`, below.